package.json

```diff
@@ -1,33 +1,32 @@
 {
-  "name": "minimalistic-react-form",
+  "name": "simple-react-forms",
   "version": "1.0.0",
   "description": "Simple form implementation using react",
-  "main": "index.js",
+  "main": "lib/index.js",
   "scripts": {
     "start": "NODE_ENV=development node server.js",
-    "prepublish": "NODE_ENV=production webpack",
+    "prepublish": "NODE_ENV=production node_modules/.bin/webpack",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/mathanpec/minimalistic-react-form.git"
+    "url": "git+https://github.com/mathanpec/simple-react-forms.git"
   },
   "keywords": [
     "react-form",
     "react-forms",
-    "simple-react-form",
-    "react-form-component"
+    "simple-react-forms",
+    "react-form-component",
+    "react-component"
```

Comment:
This will make sure your component shows up at [http://react-components.com/](http://react-components.com/)